src/core/CheckBox/CheckBox/styles.ts

```diff
@@ -0,0 +1,63 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    '&:hover': {
+      backgroundColor: 'transparent',
+    },
+    '& span': {
+      borderRadius: '0.1875rem',
+      boxShadow: 'none',
+      backgroundColor: 'transparent',
+    },
+  },
+  icon: {
+    width: '1.5rem',
+    height: '1.5rem',
+    borderWidth: '0.03125rem',
+    borderStyle: 'solid',
+    borderColor: theme.palette.border.main,
+    'input:hover ~ &': {
+      borderColor: theme.palette.secondary.main,
+    },
+    'input:disabled ~ &': {
+      borderColor: theme.palette.disabledBackground,
+    },
+  },
+  checkedIcon: {
+    width: '1.5rem',
+    height: '1.5rem',
+    borderWidth: '0.03125rem',
+    borderStyle: 'solid',
+    borderColor: theme.palette.secondary.main,
+    'input:disabled ~ &': {
+      borderColor: theme.palette.disabledBackground,
+      '&:before': {
+        display: 'block',
+        width: '1.375rem',
+        height: '1.375rem',
+        backgroundImage:
```

Comment:
CheckBox has an icon API property if if you can use that to make the code shorter
https://material-ui.com/api/checkbox/